README.md

```diff
@@ -150,4 +97,4 @@ and run the following commands:
 > .\Debug\ripplelibppdemo.exe
```

Comment:
The README needs to be updated for Windows builds to describe the patch that you made to appveyor.yml, and to update the target to `Visual Studio 15 2017 Win64".
```
       (Get-Content extras/rippled/CMakeLists.txt).replace('exclude_if_included (rocksdb)', '##') | Set-Content extras/rippled/CMakeLists.txt
```

Reply:
good point - or we just put this on hold until a fix lands in `develop` (or `master`) and switch to it once the dirty hack is no longer needed. I'm open to either approach, although I hate asking users to hack around bugs.

Reply:
I'd rather wait than ship broken code....

Could the patch be scripted into CMake, the removed when the fix gets merged?